libutils/file_lib.c

```diff
@@ -36,6 +36,7 @@
 
 #ifndef __MINGW32__
 #include <glob.h>
+#include <Windows.h>
```

Comment:
Should be in an else, i.e. not for mingw32

Reply:
Fixed.